scripts/train.py

```diff
@@ -387,4 +424,8 @@ def dummy_init_fn(module: torch.nn.Module) -> None:
         raise OLMoCliError(f"Usage: {sys.argv[0]} [CONFIG_PATH] [OPTIONS]")
 
     cfg = TrainConfig.load(yaml_path, [clean_opt(s) for s in args_list])
+    if mps_device:
+        log.info("Device is MPS. Updating config...")
+        cfg.model.init_device = "mps"
+        cfg.distributed_strategy = "single"  # type: ignore
```

Comment:
We also need these kinds of overrides for the case of the CPU. I think it will be easier if instead of having a flag like `mps_device`, you check what's in the torch default device.